### PR TITLE
Update office-365-network-mac-perf-overview.md

### PR DESCRIPTION
**Pull request by ivybarley**
@jwhit-MSFT In this PR, I added a Note about the use of IPv6 and IPv4 addresses. @pandrew1

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/office-365-network-mac-perf-overview.md [hunk 1]
```diff
@@ -70,7 +70,14 @@ For this option, you need to identify a person at each location. Ask them to bro
 
 Test reports are linked to a location if it was added with LAN subnet information, otherwise they are shown at the city location only.
 
+<<<<<<< Updated upstream
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/office-365-network-mac-perf-overview.md]:
```suggestion
```

File: microsoft-365/enterprise/office-365-network-mac-perf-overview.md [hunk 2]
```diff
@@ -70,7 +70,14 @@ For this option, you need to identify a person at each location. Ask them to bro
...
 Measurement samples and office locations should start to appear 2-3 minutes after a test report is completed. For more information, see [Microsoft 365 network connectivity test (preview)](office-365-network-mac-perf-onboarding-tool.md).
+=======
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/office-365-network-mac-perf-overview.md]:
```suggestion
```

File: microsoft-365/enterprise/office-365-network-mac-perf-overview.md [hunk 3]
```diff
@@ -70,7 +70,14 @@ For this option, you need to identify a person at each location. Ask them to bro
...
+Measurement samples and office locations should start to appear 2-3 minutes after a test report is completed. For more information, see [Microsoft 365 network connectivity test](office-365-network-mac-perf-onboarding-tool.md).
+
+>[!NOTE]
+>When adding your office locations to Microsoft 365 network connectivity in the admin center, you can provide either IPv4 or IPv6 addresses for your LAN subnets. Egress IP addresses must use IPv4.
+>>>>>>> Stashed changes
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/office-365-network-mac-perf-overview.md]:
```suggestion
> [!NOTE]
> When adding your office locations to Microsoft 365 network connectivity in the Microsoft 365 admin center, you can provide either IPv4 or IPv6 addresses for your LAN subnets. Egress IP addresses must use IPv4.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ivybarley, kindly commit the suggested changes. Many thanks.

**Comment by ivybarley** [comment]
@JohanFreelancer9 I've committed your suggestions. Cc: @pandrew1

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@jwhit-MSFT Please merge this PR. Thanks

**Comment by ivybarley** [comment]
Asking @LauraWi to merge this PR as Jason is out of office. Thanks. Cc: @pandrew1

**Pull request merged**